### DISCUSSION TITLE
Token

### DISCUSSION THREAD

**Discussion by slafarguette** [vladimirs-git/fortigate-api#12] [General]
Hi,

First  thanks for your great work.

Question :  is it possible using you api to connect with token and not user and password tuple ?

Thanks

**Comment by vladimirs-git** [comment]
Oh oh oh token :(
It was my heartache and the reason for this project. 
Some features didn't work with the token and I started this API to stop using it.
In the current version (1.1.1) token not implemented.

**Comment by slafarguette** [comment]
Thanks :)

**Comment by vladimirs-git** [comment]
token is implemented in fortigate-api 1.2.0 [./examples/fortigate_token.py](https://github.com/vladimirs-git/fortigate-api/blob/1.2.0/examples/fortigate_token.py)
